doc/guides/V8-api-for-node.md

```diff
@@ -3,29 +3,50 @@
 v8 docs, not particularly useful:
 - https://v8.dev/docs
 
+v8 docs, generated from master:
+- https://v8.paulfryzel.com/docs/master/
```

Comment:
I think these are out of date? Also Firefox warns that it's using an expired certificate.

Reply:
It's documenting V8 6.9 so yeah, pretty stale.

Reply:
Could we fork this https://github.com/paulfryzel/v8-doxygen, and redeploy it to GH pages or `Nodejs.org`

Reply:
Repo: https://github.com/gengjiawen/v8-docs

Reply:
Online preview by gitlab: https://gengjiawen.gitlab.io/-/v8-docs/-/jobs/208633573/artifacts/html/index.html.